itest/rust/src/builtin_tests/containers/packed_array_test.rs

```diff
@@ -18,6 +22,20 @@ fn packed_array_new() {
     assert_eq!(PackedByteArray::new().len(), 0);
 }
 
+#[itest]
+fn packed_array_resize_zeros_str() {
+    let mut array = PackedStringArray::new();
+    array.resize(10);
+
+    const LEN: usize = 10 * size_of::<GString>();
+    let slice = array.as_slice();
+    assert_eq!(size_of_val(slice), LEN);
+    // SAFETY: Reinterprets array data as byte array with asserted length.
+    unsafe {
+        assert_eq!(&*(slice.as_ptr() as *const [u8; LEN]), &[0u8; LEN]);
+    }
+}
```

Comment:
We should not make assumptions about the internal representations of `GString`. This is a Godot implementation detail, and adding tests like this adds a maintenance burden to godot-rust.

Reply:
Removed the assumption and corresponding test.